dc-model/src/main/java/de/digitalcollections/model/identifiable/entity/agent/CorporateBody.java

```diff
@@ -107,44 +110,45 @@ public int hashCode() {
     return Objects.hash(super.hashCode(), homepageUrl, text);
   }
 
-  public static Builder builder() {
-    return new Builder();
-  }
-
-  public static class Builder extends Agent.Builder<CorporateBody, Builder> {
+  public abstract static class CorporateBodyBuilder<
+          C extends CorporateBody, B extends CorporateBodyBuilder<C, B>>
+      extends AgentBuilder<C, B> {
 
-    @Override
-    protected EntityType getEntityType() {
-      return EntityType.CORPORATE_BODY;
+    public B text(Locale locale, String text) {
+      if (this.text == null) {
+        this.text = new LocalizedStructuredContent();
+      }
+      StructuredContent localizedDescription = this.text.get(locale);
+      if (localizedDescription == null) {
+        localizedDescription = new StructuredContent();
+      }
+      ContentBlock paragraph = StringUtils.hasText(text) ? new Paragraph(text) : new Paragraph();
```

Comment:
muss stringutils sein? bisher hatte das modell da auch noch keine abhängigkeit, oder? sollten wir mal klären, wie wir leere strings prüfen. ich meine, dass das nun auch mit java selbst gut geht. oder mit der bereits vorhandenen apache commons library: StringUtils.isBlank(string) ?

Reply:
Ist auf Bordmittel umgebaut, der Builder sollte hier in der Tat keine weitere Abhängigkeit einführen,